target/scripts/helpers/error.sh

```diff
@@ -70,5 +70,8 @@ function _shutdown
 
   _notify 'fatal' "${FATAL_ERROR_MESSAGE}"
   _notify 'err' "Shutting down.."
-  kill 1
+
+
+  kill -s SIGTERM 1
```

Comment:
`SIGTERM` is the default. I am a fan of just providing what's necessary as argument.

Reply:
How do you test this? This definitely works. Think about starting the container without a mail account. DMS keeps restarting in a loop --> stopped by `_shutdown` and then restarted via docker-compose policy `restart: always`.

Reply:
I started DMS with `docker run -it --rm mailserver/docker-mailserver` and the container does not stop. PID 1 is still alive... I was curious about this too, as I too thought `kill` would by default send `SIGTERM`.

Reply:
Does nothing happens or what is the result? What does `docker logs` shows, after executing `kill 1`.

`docker exec -it mailserver kill 1` works for me, as expected.

Reply:
I thought DMS would exit automatically if there are no accounts, but the container just does not stop. Maybe there is a misconception on my side.